code/datums/gamemodes/distress.dm

```diff
@@ -33,13 +33,18 @@
 		/datum/job/xenomorph/queen = 1
 	)
 
+	// Xeno respawns
+	xeno_respawn_wave_timer = 15 MINUTES
+	xeno_respawn_silo_reduction = 2 MINUTES
+
 	var/round_stage = DISTRESS_MARINE_DEPLOYMENT
 
-	var/bioscan_current_interval = 45 MINUTES
+	var/bioscan_current_interval = 30 MINUTES
```

Comment:
This is going to be part of a large change were we reduce the time taken to deploy and since we deploy faster, we should start getting bioscans sooner.

I just haven't changed the deployment lock yet.